instrumentation/rabbitmq-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/RabbitChannelNetAttributesGetter.java

```diff
@@ -20,7 +20,8 @@ public String transport(ChannelAndMethod channelAndMethod, @Nullable Void unused
   @Nullable
   @Override
   public String peerName(ChannelAndMethod channelAndMethod, @Nullable Void unused) {
-    return channelAndMethod.getChannel().getConnection().getAddress().getHostName();
```

Comment:
Apparently it is possible to read the cached hostname from `InetAddress` with `new InetSocketAddress(channelAndMethod.getChannel().getConnection().getAddress(), 0).getHostString()`. In this case it doesn't seem to change anything because the address really only has the ip. Added this note in case you find some other place where this could come handy.